vic/drivers/cesm/src/display_current_settings.c

```diff
@@ -379,11 +379,11 @@ display_current_settings(int mode)
     fprintf(LOG_DEST, "Input State File:\n");
     if (options.INIT_STATE) {
         fprintf(LOG_DEST, "INIT_STATE\t\tTRUE\t%s\n", filenames.init_state);
-        if (options.BINARY_STATE_FILE) {
-            fprintf(LOG_DEST, "BINARY_STATE_FILE\tTRUE\n");
+        if (options.STATE_FORMAT == ASCII) {
+            fprintf(LOG_DEST, "STATE_FORMAT\tASCII\n");
         }
-        else {
-            fprintf(LOG_DEST, "BINARY_STATE_FILE\tFALSE\n");
+        else if (options.STATE_FORMAT == BINARY) {
+            fprintf(LOG_DEST, "STATE_FORMAT\tBINARY\n");
```

Comment:
Are these really the options for the CESM driver? Wouldn't the image drivers be limited to one of the `netcdf` flavors?